routes.go

```diff
@@ -98,6 +98,10 @@ func NewRouter() *http.Server {
 		r.Delete("/channel/{id}", deleteChannel)
 	})
 
+	r.Group(func(r chi.Router) {
+		r.Post("/tribe/messageBundle", HandelTribeMessageBundleFromRelay)
+	})
+
```

Comment:
Need to move this under `r.Use(PubKeyContext)` otherwise will fail on auth

Reply:
done